.eslintrc

```diff
@@ -0,0 +1,22 @@
+{
+	"extends": "eslint-config-gyandeep",
```

Comment:
New eslint actually resolves it inside its processing.
Repo link: https://github.com/gyandeeps/eslint-config-gyandeep
More about eslint shareable config: http://eslint.org/docs/developer-guide/shareable-configs